trendico/views.py

```diff
@@ -235,3 +231,28 @@ def get(self, request):
             'wishlist_item_count': wishlist_items.count(),
             'wishlist_items': wishlist_data,
         })
+
+
+class CheckoutView(View):
+    template_name = 'checkout.html'
+
+    def get(self, request, *args, **kwargs):
+        form = OrderForm()
+        context = {'form': form}
+        return render(request, self.template_name, context)
+
+    def post(self, request, *args, **kwargs):
+        form = OrderForm(request.POST)
+        if form.is_valid():
+            order = form.save(commit=False)
+            order.user = request.user
+            order.save()
+
+            cart = Cart.objects.get(user=request.user)
+            order.products.set(cart.products.all())
```

Comment:
sometimes we have more than 10 products in our cart but we make a purchase of only one, this line fails this case, how can we make a purchase/order with selected products from cart and remove those specific products from cart? not entire cart?